src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACrossTargetedLibrary.cs

```diff
@@ -142,5 +142,46 @@ public void OutputPathDoesNotHaveDuplicatedBackslashesInOuterBuild()
             string outputPathValue = File.ReadAllText(Path.Combine(testAsset.TestRoot, testProject.Name, "OutputPathValue.txt"));
             outputPathValue.Trim().Should().NotContain("\\\\");
         }
+
+        [Fact]
+        public void OuterBuildImportsUserFile()
```

Comment:
I think this needs a `[RequiresMSBuildVersionFact("xxx")]` (with the version to be derived from whenever we check in the MSBuild side).